src/elarm_log.erl

```diff
@@ -50,11 +50,11 @@ init(_Opts) ->
 
 -spec new_alarm(alarm(), #log_state{}) -> {ok, #log_state{}}.
```

Comment:
The spec in line 51 uses a record, please define a type for the record and use that instead.